exasol_integration_test_docker_environment/lib/test_environment/shell_variables.py

```diff
@@ -0,0 +1,53 @@
+from typing import Dict
+from exasol_integration_test_docker_environment.lib.data.environment_info import EnvironmentInfo
+
+
+class ShellVariables:
+    """
+    Represents a collection of unix shell environment variables.
+    """
+    def __init__(self, env: Dict[str, str]):
+        self.env = env
+
+    @classmethod
+    def from_test_environment_info(
+            cls,
+            test_container_name: str,
```

Comment:
I think, the test_environment_info can give you the test_container_name if the container exists